WordPress/Classes/ViewRelated/Comments/CommentsTableViewCell.swift

```diff
@@ -78,38 +80,26 @@ private extension CommentsTableViewCell {
         gravatarImageView.downloadGravatarWithEmail(unwrappedEmail, placeholderImage: placeholderImage)
     }
 
-    // MARK: - Refresh UI
+    // MARK: - Configure UI
 
-    func refreshCommentLabels() {
+    func configurePendingIndicator() {
+        pendingIndicator.layer.cornerRadius = pendingIndicatorWidthConstraint.constant / 2
```

Comment:
A thought.  Since this gets computed every time the cell is configured, but it really only needs to happen once, if you want it could be configured like so via Interface Builder:
<img width="663" alt="Screen Shot 2021-02-19 at 10 48 31 AM" src="https://user-images.githubusercontent.com/1435271/108534890-46050500-72a0-11eb-829f-86b9324a483e.png">
Or probably done in awakeFromNib like
```
    open override func awakeFromNib() {
        super.awakeFromNib()
        pendingIndicator.layer.cornerRadius = pendingIndicatorWidthConstraint.constant / 2
    }
```
It's a super minor optimization tho. Maybe not worth the change.

Reply:
> this gets computed every time the cell is configured

Thanks @aerych . I didn't think of that.

The reason I did it dynamically in code is so it "just works" if the width of the indicator is changed. So I'll go with option 2 - `awakeFromNib`.